frontend/pages/admin/IntegrationsPage/cards/MdmSettings/components/EndUserMigrationSection/EndUserMigrationSection.tsx

```diff
@@ -59,7 +59,10 @@ const EndUserMigrationSection = ({ router }: IEndUserMigrationSectionProps) => {
   };
 
   const toggleMigrationEnabled = () => {
-    setFormData({ ...formData, isEnabled: !formData.isEnabled });
+    setFormData((prevFormData) => ({
```

Comment:
I think there's a few other places in this component where we're also setting formData that can be updated to this pattern for consistency.